experiments.json

```diff
@@ -1,15 +1,28 @@
 {
-  "onboarding-a": {
```

Comment:
Also, we should still keep an "onboarding-a" set to 100% of users, for versions 43-45.